language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/LSCommandExecutorException.java

```diff
@@ -0,0 +1,25 @@
+package org.ballerinalang.langserver.command;
```

Comment:
Since this is a newly added file shall we have the license header and the version?

Reply:
Fixed